src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -1426,6 +1444,7 @@ initialize_query_table()
   emplace_sysfs_get<query::xocl_errors>                        ("", "xocl_errors");
 
   emplace_func0_request<query::pcie_bdf,                       bdf>();
+  emplace_func0_request<query::pcie_id,                        id>();
```

Comment:
I think `pcie_id` would be better?
```suggestion
  emplace_func0_request<query::pcie_id,                        pcie_id>();
```